spec/unit/util/tagging_spec.rb

```diff
@@ -88,6 +88,13 @@
       expect(tagger).to be_tagged("goodbye")
     end
 
+    it "downcases tag arguments" do
+      tagger.tag("hello")
+      tagger.tag("goodbye")
+      expect(tagger).to be_tagged(:HEllO)
+      expect(tagger).to be_tagged("GooDByE")
+    end
+
```

Comment:
think test should cover what is expected if tagging is done with uc word as well